trendmicro_deepsecurity/plugin.spec.yaml

```diff
@@ -0,0 +1,99 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: trendmicro_deepsecurity
+title: Trend Micro Deep Security
+description: Utilise the Deep Security IPS engine to protect against vulnerabilities without installing a patch (virtual patching)
+version: 1.0.0
+vendor: philipp-behmer
+support: community
+status: []
+enable_cache: false
+resources:
+  source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/trendmicro_deepsecurity
+  license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+  vendor_url: https://trendmicro.com
+tags:
+- vulnerability_scanning
+- vulnerability_management
+- intrusion_detection
+- intrusion_protection
+- virtual_patching
+- remediation
+- trend_micro
+- deep_security
+hub_tags:
+  use_cases: [vulnerability_management, cloud_security, remediation_management, threat_detection_and_response ]
+  keywords: [vulnerability, scanning, management, remediation, virtual, patching, intrusion, detection, prevention, remediation, trend, micro, deep, security, trendmicro, deepsecurity]
```

Comment:
```suggestion
  keywords: [vulnerability, scanning, remediation, patching, intrusion, detection, prevention, remediation, trend micro, deep, security, trendmicro, deepsecurity]
```

Reply:
ah you are allowed to put 2 words in one keyword. good to know

Reply:
done